### ISSUE TITLE
cors block prevents reading the request body

### ISSUE THREAD

**Issue by johakoch** [avenga/couper#354]
**Describe the bug**
With a `cors` block in the couper config, couper somehow prevents reading the request body.

**To Reproduce**
1. Which Couper version? 1.5

2. Provide your configuration file `*.hcl`. Remove sensitive data.

```hcl
server "cors" {
  cors {
    allowed_origins = ["https://editor.swagger.io"]
  }

  api {
    endpoint "/reflect" {
      response {
        json_body = request
      }
    }
  }
}
```

3. Provide a `curl` call for reproduction
```
$ curl -si -H "Origin: https://editor.swagger.io" -d username=me localhost:8080/reflect
HTTP/1.1 200 OK
Access-Control-Allow-Origin: https://editor.swagger.io
Connection: close
Content-Type: application/json
Couper-Request-Id: c5o381d916bj6ocli850
Server: couper.io
Vary: Origin
Vary: Accept-Encoding
Date: Wed, 20 Oct 2021 15:21:41 GMT
Content-Length: 486

{"body":null,"context":{},"cookies":{},"form_body":{},"headers":{"accept":"*/*","content-length":"11","content-type":"application/x-www-form-urlencoded","couper-request-id":"c5o381d916bj6ocli850","origin":"https://editor.swagger.io","user-agent":"curl/7.67.0"},"host":"localhost","id":"c5o381d916bj6ocli850","json_body":{},"method":"POST","origin":"http://localhost:8080","path":"/reflect","path_params":{},"port":8080,"protocol":"http","query":{},"url":"http://localhost:8080/reflect"}
```

4. See an error response or error log.
 --

**Expected behavior**
The value of `body` should be `"username=me"`, the value of `form_body` should be `{"username":["me"]}`, just like it is without the `cors` block.

**Comment by malud** [comment]
This may have been solved due to some internal changes in #317 . Can you confirm this?

**Comment by johakoch** [comment]
Fixed, thank you.
